docs/quickstart-cli.md

```diff
@@ -193,6 +187,25 @@ If you've mounted the USB stick with a file manager, this command should work to
 sudo cp initial-registration.yaml /run/media/$USER/COS_LIVE/livecd-cloud-config.yaml
 ```
 
+If you prefer using some CLI tools:
+
+```shell showLineNumbers
+IMAGE=rpi.raw
+DEST=$(mktemp -d)
+
+SECTORSIZE=$(sfdisk -J ${IMAGE} | jq '.partitiontable.sectorsize')
+DATAPARTITIONSTART=$(sfdisk -J ${IMAGE} | jq '.partitiontable.partitions[1].start')
+sudo mount -o rw,loop,offset=$((${SECTORSIZE}*${DATAPARTITIONSTART})) ${IMAGE} ${DEST}
+sudo cp initial-registration.yaml ${DEST}/livecd-cloud-config.yaml
+sudo umount ${DEST}
```

Comment:
```suggestion
sudo umount ${DEST}; rmdir ${DEST}
```

Reply:
Added as a new line, thanks for the suggestion!